atomic_admin.gemspec

```diff
@@ -0,0 +1,26 @@
+require_relative "lib/atomic_admin/version"
+
+Gem::Specification.new do |spec|
```

Comment:
There are lots of example values here. I can't remember if we decided on making this open source or not?